modules/secret/output.tf

```diff
@@ -0,0 +1,4 @@
+output "this" {
+  value     = opslevel_secret.this
+  sensitive = true
```

Comment:
why is this sensitive?

Reply:
Because it's `value` field is marked sensitive. I don't remember if it's required or convention but we also do this for the azure integration resource and the gcp integration resource

Reply:
ok